include/sys/fs/zfs.h

```diff
@@ -741,6 +744,24 @@ typedef enum zpool_errata {
 	ZPOOL_ERRATA_ZOL_2094_ASYNC_DESTROY,
 } zpool_errata_t;
 
+#ifndef BIT /* Include BIT() for userspace */
+#define	BIT(n)	(1ULL << (n))
+#endif
+
+enum vdev_stat_flags {VS_DEFAULT = 0, VS_LATENCY = 1, VS_QUEUES = 2,
```

Comment:
The following would be consistent with the existing style and naming conventions.  You might also consider dropping the BIT() macro, it's not clear to me that is actually makes thing more readable.

```
typedef enum vdev_stat_flags {
        VS_DEFAULT_BIT  = 0,
        VS_LATENCY_BIT  = 1,
        VS_QUEUES_BIT   = 2,
        VS_HISTO_BIT    = 3,
        VS_FLAGS_COUNT  = 4,
} vdev_stat_flags_t;

/* Mask versions of all our flags */
#define VS_DEFAULT      BIT(VS_DEFAULT_BIT)
#define VS_LATENCY      BIT(VS_LATENCY_BIT)
#define VS_QUEUES       BIT(VS_QUEUES_BIT)
#define VS_HISTO        BIT(VS_HISTO_BIT)

#define VS_FLAGS_ALL (VS_LATENCY_BIT | VS_QUEUES_BIT | VS_HISTO_BIT | VS_FLAGS_COUNT);
```